src/Core/Entities/Secret.cs

```diff
@@ -21,12 +22,14 @@ public class Secret : ITableObject<Guid>
 
     public DateTime? DeletedDate { get; set; }
 
+    [NotMapped]
+    public Dictionary<System.Guid, string>? ProjectGuids { get; set; }
```

Comment:
This is never used?

Reply:
It is used, that is being set on GetManyByOrganizationIdAsync inside SecretRepository.

Reply:
It is being used 🤔